File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    
    for(let [i,j,k] of commands){
        const splitArr=array.slice(i-1,j);
        splitArr.sort((a,b)=>a-b);
        answer.push(splitArr[k-1]);
    }
    return answer;
}

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        int idx=0;
        for(int i=0; i<commands.length; i++){
            int start=commands[i][0];
            int end=commands[i][1];
            int target=commands[i][2];
            
            int[] arr=new int[end-start+1]; 
            int q=0;
            for(int j=start-1; j<end; j++){
                arr[q++]=array[j];
            }
            Arrays.sort(arr);
            answer[idx++]=arr[target-1];
        }
        return answer;
    }
}